static/src/javascripts-legacy/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -72,10 +74,34 @@ define([
         return options.useTargetingTool ? targetingTool.isAbTestTargeted(options) : true;
     }
 
+    // Returns an array containing:
+    // - the first element matching insertBeforeSelector
+    // - all elements matching insertBeforeSelectorAll
+    function getTargets(insertBeforeSelector, insertBeforeSelectorAll) {
```

Comment:
It might be better to pass a single `String` as the selector, and then a `Boolean` flag to say if it should try and get multiple `Elements` for that selector?